Chatkit.xcodeproj/xcshareddata/xcschemes/ALL Tests.xcscheme

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We could potentially add all 3 test targets to `Chatkit` run scheme and have individual run schemes for each test target. An open question is whether we should run `System Tests` on our CI for every build or should this be triggered manually (or scheduled)? What do you think?

Reply:
Yes that is an option.  I'm not sure I have formed a solid opinion on whether System tests should run on CI.   I think it might be good to run them but then we'd need to be concerned with the setup of the Instance they run against and also the possibility of flakey tests.  

At the very least we should *compile* the System tests on CI to ensure any refactoring hasn't broken compilation.

Reply:
OK, finally got the "Swift Environment Variables Injector" working.  I have updated the `Chatkit` scheme to run all the tests and removed the `ALL Tests` scheme.  Builds are now passing on Bitrise.